nano/node/cli.cpp

```diff
@@ -88,7 +90,8 @@ void nano::add_node_flag_options (boost::program_options::options_description &
 		("disable_wallet_bootstrap", "Disables wallet lazy bootstrap")
 		("disable_bootstrap_listener", "Disables bootstrap processing for TCP listener (not including realtime network TCP connections)")
 		("disable_tcp_realtime", "Disables TCP realtime network")
-		("disable_udp", "Disables UDP realtime network")
+		("disable_udp", "(Deprecated) UDP is disabled by default")
+		("enable_udp", "Enables UDP realtime network")
```

Comment:
What do you think of making an option like "enable_transport" that's a program_options::multitoken with tcp and udp as options. So it would be "--enable_transport = tcp udp" or "--enable_transport=tcp --enable_transport=udp"

The example they have for setting this up is
("email", value<string>()->multitoken(), "email to send to")
https://www.boost.org/doc/libs/1_71_0/doc/html/program_options/overview.html#id-1.3.32.5.7

Reply:
Do we want to disable tcp by default as well?

Reply:
I guess we wouldn't want to force the transport to always be chosen.